senders/proxy.go

```diff
@@ -133,6 +135,15 @@ func makeConnHandler(host string, port, flushIntervalSeconds int, prefix string,
 	return internal.NewProxyConnectionHandler(addr, flushInterval, prefix, internalRegistry)
 }
 
+func SetTag(key, value string) internal.RegistryOption {
```

Comment:
We need to communicate to SDK users that these tags only apply to internal SDK metrics. I suggest we try just adding a field to ProxyConfiguration of `SDKMetricsTags map[string]string`. I think that would also allow rolling back the change of making `MetricRegistry.Tags` public.

Reply:
Also, can you look at adding the same feature to the sender in `client.go`? That's our non-deprecated sender, and I'd like to keep feature parity.

Reply:
Sure, thanks @oppegard